### PR TITLE
vtl context only gets headers from request

### LINKED ISSUE #96: Converting circular structure to JSON on $utils.toJson($context) at ....
Since this commit: acad4497600e071eae56765a2dcfa671315c66a5
I am getting this error when doing
`$utils.toJson($context)`.

This happens I believe because there must have some circular reference in the request object https://github.com/ConduitVC/aws-utils/blob/master/packages/appsync-emulator-serverless/serverCore.js#L280

@JimmyHurrah

### PR DESCRIPTION
**Pull request by cbaron**
This should fix #96

### CODE REVIEW

**Comment by cbaron** [comment]
@bboure -- let me know if this fixes your problem

**Comment by cbaron** [comment]
cc @JimmyHurrah

File: packages/appsync-emulator-serverless/schema.js [hunk 1]
```diff
@@ -47,14 +47,14 @@ class AppSyncError extends Error {
 const buildVTLContext = ({ root, vars, context, info }, result = null) => {
   const {
     jwt: { iss: issuer, sub },
-    request,
+    request: { headers },
```
Comment (lightsofapollo) [inline packages/appsync-emulator-serverless/schema.js]: Looks like request is not always set (probably somewhere in tests) I would harden this to accept a null object.

**Comment by JimmyHurrah** [comment]
Yeah, this seems like the right thing to do. It was a bit reckless of me to pass the whole request object. Sorry about that 🤕

**Pull request merged**